app/views/govuk_publishing_components/components/docs/action_link.yml

```diff
@@ -49,6 +49,27 @@ examples:
       light_text: true
     context:
       dark_background: true
+  mobile_subtext:
+    description: Lock the subtext beneath the main text, so the component with subtext always appears as it does on mobile.
+    data:
+      text: "Try not to panic now"
```

Comment:
The double quotes shouldn't be needed if the string contain no special characters

Reply:
Thanks, have addressed this.